### PR TITLE
refactor(flows): update copy to improve usability

### PR DESCRIPTION
**Pull request by alexpaxton**
- Minimap closed by default
- First cell can be deleted
- Update copy in empty script cells to be more instructive
- Update copy in empty visualization cells

Screen Shot 2020-06-15 at 1 52 59 PM

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/notebooks/components/panel/NotebookPanel.tsx [hunk 1]
```diff
@@ -42,7 +42,6 @@ const NotebookPanelHeader: FC<HeaderProps> = ({index, controls}) => {
   const {pipes, removePipe, movePipe} = useContext(NotebookContext)
   const canBeMovedUp = index > 0
   const canBeMovedDown = index < pipes.length - 1
-  const canBeRemoved = index !== 0
```
Comment (drdelambre) [inline ui/src/notebooks/components/panel/NotebookPanel.tsx:45]: WHOA.. bold move

File: ui/src/notebooks/pipes/Query/index.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ register({
     activeQuery: 0,
     queries: [
       {
-        text: '',
+        text: '// Write Flux script here',
```
Comment (drdelambre) [inline ui/src/notebooks/pipes/Query/index.ts:16]: do we want to write this for all of them, or just the first one?

File: ui/src/shared/reducers/app.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ const initialState: AppState = {
     showTemplateControlBar: false,
     timeZone: 'Local',
     navBarState: 'collapsed',
-    notebookMiniMapState: 'expanded',
+    notebookMiniMapState: 'collapsed',
```
Comment (drdelambre) [inline ui/src/shared/reducers/app.ts:32]: noice

File: ui/src/notebooks/pipes/Query/index.ts (hunk 2 above)
Comment (alexpaxton) [reply ui/src/notebooks/pipes/Query/index.ts:16]: I figure all, cuz it doesn't hurt

**Pull request merged**